cypress/e2e/Lexplore_testing/addBook.cy.js

```diff
@@ -0,0 +1,40 @@
+/// <reference types="cypress" />
+
+describe('My first e2e testing', () => {
+    beforeEach(() => {
+        cy.intercept('https://readingservicesdev.lexplore.com/activities/lastPerBook')
+        .as('interceptLogin')
+        cy.visit('https://logindev.lexplore.com/go/5yU3DaVNH0GmIjPBMJXWDg')
+        cy.wait('@interceptLogin')
+    })
+        it('Test valid add of book ', () => {
+            const cookie1 = cy.getCookie('devSessionId').should('exist')
+            const cookie2 = cy.getCookie('devSysApi').should('exist')
+            cy.setCookie(`devSessionId`, `${cookie1}`).should('exist')
+            cy.setCookie('devSysApi', `${cookie2}`).should('exist')
+            cy.request({
+                method: 'PUT',
+                failOnStatusCode: false,
```

Comment:
add book, 1 api test i to ne koristis funkciju za api nego si samo nabacao stvari u spec...